### PR TITLE
optionally verify directories at startup

### PR DESCRIPTION
**Pull request by uwiger**
Using the setup application env var `{verify_directories, false}`, setup can be told not to verify directories at startup. This can be desirable if those directories aren't being used.

### CODE REVIEW

**Comment by surik** [comment]
Are you going to merge this?

File: src/setup.erl [hunk 1]
```diff
@@ -767,8 +780,8 @@ path_entries(A, Path) ->
 make_appup_script(A, OldVsn, NewPath) ->
     {application, _, NewAppTerms} = NewApp =
         read_app(filename:join(NewPath, atom_to_list(A) ++ ".app")),
-    OldAppTerms = application:get_all_key(A),
-    _OldApp = {application, A, OldAppTerms},
+    %% OldAppTerms = application:get_all_key(A),
```
Comment (tolbrino) [inline src/setup.erl]: Is this still needed for debugging purposes?

File: src/setup.erl [hunk 2]
```diff
@@ -858,7 +871,7 @@ run_setup(Parent, Args) ->
     end.
 
 run_setup_(Parent, _Args) ->
-    Res = rpc:multicall(?MODULE, verify_directories, []),
+    Res = maybe_verify_directories(),
```
Comment (tolbrino) [inline src/setup.erl]: What is the reason for not using `multicall` here?

Comment (uwiger) [inline src/setup.erl]: Is multicall really needed? Should it be another option?

File: src/setup.erl (hunk 1 above)
Comment (uwiger) [inline src/setup.erl]: No

File: src/setup.erl (hunk 2 above)
Comment (tolbrino) [inline src/setup.erl]: It was the only spot where `multicall` was used. So just to be consistent I guess it is okay to remove it and think about whether a multi-node enforement of anything by setup makes sense as an option.

Comment (uwiger) [inline src/setup.erl]: Yeah, I believe we actually had use for it in a project you and I worked together, but I'm willing to drop it and wait for the requirement to resurface. :)

**Comment by uwiger** [comment]
Tino, you ok with merging this?

File: src/setup.erl (hunk 2 above)
Comment (tolbrino) [inline src/setup.erl]: Right, I remember the use case :-)

**Comment by tolbrino** [comment]
yes, done so

**Pull request merged**